Libraries/Microsoft.Teams.Extensions/Microsoft.Teams.Extensions.Context/ContextExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.Teams.Apps;
+using Microsoft.Teams.Api.Activities;
+using Microsoft.Graph;
+
+namespace Microsoft.Teams.Extensions.Context;
+
+public static class ContextExtensions
+{
+    /// <summary>
+    /// Get the user's graph client from the context.
+    /// </summary>
+    public static GraphServiceClient? GetUserGraphClient<TActivity>(this IContext<TActivity> context) where TActivity : IActivity
```

Comment:
I imagine as time goes on we may have more context extensions. What if someone wants the graph context extension but not the others? To provide maximum flexibility and freedom of choice you may want to change this package to `Microsoft.Teams.Extensions.Graph`

Reply:
Good idea, updated the code